sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/strategies/MessageListenerStrategy.java

```diff
@@ -159,16 +158,17 @@ protected void addOffsetToCommitOnClose(ConsumerRecord<K, V> consumerRecord) {
    * @param consumer the consumer to communicate with Kafka
    */
   public void commitOnClose(KafkaConsumer<K, V> consumer) {
-    try {
-      if (!offsetsToCommitOnClose.isEmpty()) {
-        LOGGER.info("Committing offsets on close: {}", offsetsToCommitOnClose);
-        consumer.commitAsync(offsetsToCommitOnClose, null);
-      } else {
-        LOGGER.warn("Committing offsets of last poll on close");
-        consumer.commitAsync();
-      }
-    } catch (CommitFailedException e) {
-      LOGGER.error("Commit failed", e);
+    if (!offsetsToCommitOnClose.isEmpty()) {
+      LOGGER.info("Committing offsets on close: {}", offsetsToCommitOnClose);
+      consumer.commitAsync(
```

Comment:
Do we have to catch the `FencedInstanceIdException` as stated in the method documentation?

Reply:
As we did not care about it before, I would argue, we don't need it now.

Reply:
Makes sense 😆

Reply:
We may catch and log a RuntimeException [here](https://github.com/SDA-SE/sda-dropwizard-commons/blob/5601b101625267ba905bfbe89c7e8e6051bcbc2c/sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/MessageListener.java#L105). As this code is executed in a separate thread, the information would get lost otherwise.